nncf/quantization/algorithms/min_max/algorithm.py

```diff
@@ -14,6 +14,8 @@
 from copy import deepcopy
 from typing import Any, Dict, List, Optional, OrderedDict, Set, TypeVar
 
+import numpy as np
```

Comment:
Can we refuse of using numpy here?

Reply:
What's wrong with Numpy?

Reply:
Agree, numpy is the all-needed backend package. There is no issue with that.